core/src/test/scala/org/apache/spark/deploy/worker/WorkerSuite.scala

```diff
@@ -17,13 +17,18 @@
 
 package org.apache.spark.deploy.worker
 
-import org.apache.spark.{SparkConf, SparkFunSuite}
-import org.apache.spark.deploy.Command
-
 import org.scalatest.Matchers
 
+import org.apache.spark.deploy.DeployMessages.{DriverStateChanged, ExecutorStateChanged}
+import org.apache.spark.deploy.master.DriverState
+import org.apache.spark.rpc.{RpcAddress, RpcEnv}
+import org.apache.spark.{SecurityManager, SparkConf, SparkFunSuite}
+import org.apache.spark.deploy.{ExecutorState, Command}
```

Comment:
This should be `import org.apache.spark.deploy.{Command, ExecutorState}` .
Also, could you sort these import statements in dictionary order?